packages/webpack/loader.js

```diff
@@ -31,11 +31,17 @@ async function loader(source, inputSourceMap) {
         options ||
         Object.assign(
             {
-                getFilepath: filepath => {
-                    const hash = `${utils.getFileHash(filepath)}_${++index}`;
-                    const filename = `${hash}.css`;
+                modules: false,
```

Comment:
I believe that strict boolean option is better here than something like `extension: '.module.css'`.
and if it's important to control the naming, so there are `getFilename` and `getFilepath` options

Reply:
As long as these options 
- do not interfere
- do not add confusion to the end package consumers

I think it's okay and convenient, since next.js/CRA and others use this option name as well.